doc/extensions/myexampleextension/azext_myexampleextension/__init__.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I could but that introduces a dependency between the license verify script and this file inside of docs.
Let me know if having the license header here is enough of an issue for me to make that change.